buildconfig/BuildConfig.cpp.in

```diff
@@ -76,7 +76,8 @@ Config::Config()
 
     // Assume that builds outside of Git repos are "stable"
     if (GIT_REFSPEC == QStringLiteral("GITDIR-NOTFOUND")
-        || GIT_TAG == QStringLiteral("GITDIR-NOTFOUND"))
+        || GIT_TAG == QStringLiteral("GITDIR-NOTFOUND")
+        || GIT_TAG == QStringLiteral("GIT-NOTFOUND"))
```

Comment:
This should probably also check if `GIT_REFSPEC` is empty

Reply:
true, i just didn't want to add more ugly ORs 🥸